.github/workflows/run_forklift_build_on_kind.yml

```diff
@@ -46,6 +46,8 @@ jobs:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - run: build_and_setup_everything_bazel.sh
 
+      - run: openstack/run_packstack_docker.sh
```

Comment:
we chatted about this offline - would be better to deploy it only within the job that executes the OSP suite and instead of executing a separate script, to customize the deployment so we wouldn't deploy fakeovirt+imageio/vcsim when it's not needed